server/controllers/shared/RTLConf.ts

```
@@ -99,40 +99,33 @@ export const getFile = (req, res, next) => {
-      });
+  const appConfData = common.removeSecureData(JSON.parse(JSON.stringify(common.appConfig)));
+  appConfData.allowPasswordUpdate = common.appConfig.allowPasswordUpdate;
+  appConfData.enable2FA = common.appConfig.enable2FA;
+  appConfData.selectedNodeIndex = (req.session.selectedNode && req.session.selectedNode.index ? req.session.selectedNode.index : common.selectedNode.index);
+  common.appConfig.selectedNodeIndex = appConfData.selectedNodeIndex;
+  const token = req.headers.authorization ? req.headers.authorization.split(' ')[1] : '';
+  jwt.verify(token, common.secret_key, (err, user) => {
+    if (err) {
+      // Delete unnecessary data for initial response (without security token)
+      const selNodeIdx = appConfData.nodes.findIndex((node) => node.index === appConfData.selectedNodeIndex) || 0;
+      delete appConfData.SSO.rtlCookiePath;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)